spec.bs

```diff
@@ -412,22 +421,17 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
  1. [=Associate the issuer=] |issuer| with |topLevel|.
  1. If the [=number of tokens=] for |issuer| is at least 500, return.
  1. Let |commitments| be the result of [=look up the key commitments|looking up the key commitments=] for |issuer|.
- 1. If |commitments| is null,  return.
- 1. Pass issuer public keys to cryptographic procedures. Reject if keys are
-         malformed.
-
-    Issue(187): How do we get the issuer public keys here?
-
- 1. [=Discard tokens=] from |issuer| that are signed with keys other than those from
+ 1. If |commitments| is null, return.
+ 1. Perform [=discard tokens=] with |issuer| that are signed with keys other than those from
             the issuer's most recent commitments.
 
     Issue(189): This probably needs to be its own algorithm, how is "most recent" defined here?
 
  1. Let |numTokens| be |issuer|'s [=max batch size=] or an [=implementation-defined=] limit on the number of tokens (which is recommended to be 100), whichever is smaller.
 
- 1. Let |tokens| be the result of [=generate masked tokens|generating masked tokens=] with |numTokens|.
+ 1. Let (|issueHeader|, |pretokens|) be the result of [=generate masked tokens|generating masked tokens=] with |commitments| and |numTokens|.
  1. Set |request|'s [=cache mode=] to `"no-store"`.
- 1. Let |base64-encoded-tokens| be the base64-encoded version of |tokens|.
+ 1. Let |base64-encoded-tokens| be the base64-encoded version of |issueHeader|.
```

Comment:
This isn't new and the entire spec is pretty inconsistent with casing so far, but this is the first time I'm seeing this casing style (dashes?). Should probably rename it to either `[=base64-encoded tokens=]` or `[=base64EncodedTokens=]`, depending on our overall preference for the document.